src/forwardAttachment.js

```diff
@@ -0,0 +1,43 @@
+"use strict";
+
+var utils = require("../utils");
+var log = require("npmlog");
+
+module.exports = function(defaultFuncs, api, ctx) {
+  return function forwardAttachment(attachmentID, userOrUsers, callback) {
+    if (!callback) {
+      callback = function() {};
+    }
+
+    var form = {
+      attachment_id: attachmentId,
+    };
+
+    if(utils.getType(userOrUsers) !== "Array") {
+      userOrUsers = [userOrUsers];
+    }
+    
+    var timestamp = Math.floor(Date.now() / 1000);
+
+    for (var i = 0; i < userOrUsers.length; i++) {
+      //That's good, the key of the array is really timestmap in seconds + index
+      //Probably time when the attachment will be sent?
+      form['recipient_map[' + (timestamp + i) + ']'] = userOrUsers[i];
+    }
+
+    defaultFuncs
+      .post("https://www.messenger.com/mercury/attachments/forward/?dpr=1", ctx.jar, form)
```

Comment:
I would imagine you can remove that `?dpr=1` param. Could you double check if it works without?